vktrace/vktrace_replay/vkreplay_vkreplay.cpp

```diff
@@ -2256,16 +2112,182 @@ VkResult vkReplay::manually_replay_vkAllocateMemory(packet_vkAllocateMemory *pPa
         }
     }
 
-    if (!m_objMapper.m_adjustForGPU) {
-        uint32_t replayIdx;
-        if (getMemoryTypeIdx(pPacket->device, remappedDevice, pPacket->pAllocateInfo->memoryTypeIndex, &replayIdx)) {
-            *((uint32_t *)&pPacket->pAllocateInfo->memoryTypeIndex) = replayIdx;
+    if ((uint64_t)pPacket->pMemory == (uint64_t)0x22a5caeb580) {
+        pPacket->pMemory = pPacket->pMemory;
```

Comment:
That is debug code that was not removed. I used it to set a breakpoint when a certain vkAllocateMemory packet was processed. Will be removed.